src/commands/readme.js

```diff
@@ -8,10 +8,18 @@ const sharedOptions = require('./shared_options');
 const inject = require('mdast-util-inject');
 const chalk = require('chalk');
 const disparity = require('disparity');
+const getReadmeFile = require('../get-readme-file');
 
 module.exports.command = 'readme [input..]';
 module.exports.description = 'inject documentation into your README.md';
 
+let defaultReadmeFile = 'README.md';
+try {
+  defaultReadmeFile = getReadmeFile('.');
+} catch (err) {
+  // ignore and use default README.md
```

Comment:
Seems unnecessary - we shouldn't expect getReadmeFile to throw, right? This could be 

```js
const defaultReadmeFile = getReadmeFile('.') || 'README.md';
```